modules/internal/DefaultAssociative.chpl

```diff
@@ -430,14 +430,14 @@ module DefaultAssociative {
       }
     }
   
-    iter dsiSorted() {
+    iter dsiSorted(comparator) {
```

Comment:
Would it be valuable to support comparators as an optional arg, or does it not matter since it's an internal iter anyway?